scene/2d/tile_map.cpp

```diff
@@ -3086,7 +3086,7 @@ void TileMap::_build_runtime_update_tile_data(SelfList<TileMapQuadrant>::List &r
 
 							// Create the runtime TileData.
 							TileData *tile_data_runtime_use = tile_data->duplicate();
-							tile_data->set_allow_transform(true);
+							tile_data_runtime_use->set_allow_transform(true);
```

Comment:
@groud That's unrelated changed, but I think it's the duplicated TileData which were supposed to be allowed to be transformed? :thinking:

Reply:
Yes, that sounds good.

Reply:
I added the change to https://github.com/godotengine/godot/pull/78328 too. I don't know if we want to keep it in this PR. I guess it depends on the first merged. ^_^"